CRM/Eck/DAO/Entity.php

```diff
@@ -19,6 +19,7 @@
  *
  */
 class CRM_Eck_DAO_Entity extends CRM_Core_DAO {
+  const EXT = E::LONG_NAME;
```

Comment:
It's just a standard part of any "normal" DAO, so I thought it would be a good idea for this "fake" DAO to emulate a normal one as much as possible to avoid future problems :)